src/wallet/wallet.cpp

```diff
@@ -3908,7 +3908,7 @@ bool CWallet::CreateCoinStake(const CBlockIndex* pindexBest, unsigned int nBits,
                 LOCK(cs_main);
                 //Double check that this will pass time requirements
                 if (nTxNewTime <= nTimeMinBlock) {
-                    LogPrint(BCLog::BLOCKCREATION, "CreateCoinStake() : kernel found, but it is too far in the past \n");
+                    LogPrint(BCLog::STAKING, "CreateCoinStake() : kernel found, but it is too far in the past \n");
```

Comment:
NIT While you're here; change this to a `"%s: ` with `__func__`

Reply:
Done.